modules/ROOT/pages/subqueries/call-subquery.adoc

```diff
@@ -1,53 +1,63 @@
+:description: This page describes how to use Cypher's `CALL` subquery.
 
-The `CALL` clause can be used to invoke a subquery.
-Unlike other subqueries in Cypher, it can be used to perform changes to the database (e.g. xref:clauses/create.adoc[] new nodes), and it requires an importing xref:clauses/with.adoc[] clause.
+The `CALL` clause can be used to invoke subqueries that execute operations within a defined scope, thereby optimizing data handling and query efficiency. 
+Unlike other subqueries in Cypher, `CALL` subqueries can be used to perform changes to the database (e.g. xref:clauses/create.adoc[] new nodes).
 
 [NOTE]
-====
 The `CALL` clause is also used for calling procedures.
-For descriptions of the `CALL` clause in this context, refer to xref::clauses/call.adoc[`CALL` procedure].
-====
+For descriptions of the `CALL` clause in this context, refer to the xref::clauses/call.adoc[`CALL` procedure].
 
 [[call-example-graph]]
 == Example graph
 
-The following graph is used for the examples below:
+A graph with the following schema is used for the examples below:
 
 image::call_subquery_graph.svg[]
 
 To recreate the graph, run the following query in an empty Neo4j database:
 
 [source, cypher, role=test-setup]
 ----
-CREATE
- (a:Person:Child {name: 'Alice', age: 20}),
- (b:Person {name: 'Bob', age: 27}),
- (c:Person:Parent {name: 'Charlie', age: 65}),
- (d:Person {name: 'Dora', age: 30})
- CREATE (a)-[:FRIEND_OF]->(b)
- CREATE (a)-[:CHILD_OF]->(c)
- CREATE (a)-[:OWES {dollars: 20}]->(c)
- CREATE (a)-[:OWES {dollars: 25}]->(b)
- CREATE (b)-[:OWES {dollars: 35}]->(d)
- CREATE (d)-[:OWES {dollars: 15}]->(b)
- CREATE (d)-[:OWES {dollars: 30}]->(b)
-CREATE (:Counter {count: 0})
+CREATE (teamA:Team {name: 'Team A'}),
+       (teamB:Team {name: 'Team B'}),
+       (teamC:Team {name: 'Team C'}),
+       (playerA:Player {name: 'Player A', age: 21}),
+       (playerB:Player {name: 'Player B', age: 23}),
+       (playerC:Player {name: 'Player C', age: 19}),
+       (playerD:Player {name: 'Player D', age: 30}),
+       (playerE:Player {name: 'Player E', age: 25}),
+       (playerF:Player {name: 'Player F', age: 35}),
+       (playerA)-[:PLAYS_FOR]->(teamA),
+       (playerB)-[:PLAYS_FOR]->(teamA),
+       (playerC)-[:PLAYS_FOR]->(teamA),
+       (playerD)-[:PLAYS_FOR]->(teamB),
+       (playerE)-[:PLAYS_FOR]->(teamC),
+       (playerF)-[:PLAYS_FOR]->(teamC),
+       (playerA)-[:FRIEND_OF]->(playerB),
+       (playerA)-[:FRIEND_OF]->(playerC),
+       (playerB)-[:FRIEND_OF]->(playerF),
+       (playerC)-[:FRIEND_OF]->(playerD),
+       (teamA)-[:OWES {dollars: 1500}]->(teamB),
+       (teamA)-[:OWES {dollars: 3000}]->(teamB),
+       (teamB)-[:OWES {dollars: 1700}]->(teamC),
+       (teamC)-[:OWES {dollars: 5000}]->(teamB)
 ----
 
-[[call-semantics]]
-== Semantics
+== Semantics and performance
 
 A `CALL` subquery is executed once for each incoming row.
+The variables declared in a subquery are available to the outer scope of the enclosing query.
```

Comment:
Find this a little difficult to understand. All returning variables of the subquery is available to the outer scope but any other declaration within the subquery is not in scope of the outer query.
e.g. variable `n` is not available in outer scope but `innerReturn` is
```
UNWIND [0, 1, 2] AS x
CALL () {
  MATCH (n)
  RETURN 'hello' AS innerReturn
}
RETURN innerReturn
```